src/widgets/AdminChooseDates.tsx

```diff
@@ -15,9 +15,8 @@ export function AdminChooseDates({ value, onChange }: AdminChooseDatesProps) {
 
 	const handleDateChange = (newDate: Date | undefined) => {
 		if (newDate) {
-			const adjustedDate = new Date(newDate.getTime() + 60 * 60 * 1000);
-			console.log("Adjusted Date (UTC+1):", adjustedDate.toISOString());
-			onChange(adjustedDate.toISOString());
+			const adjustedDate = new Date(newDate.getTime());
+			onChange(adjustedDate.toLocaleString());
```

Comment:
I'm probably missing something, but why do you construct a new Date from an existing date instead of just calling
```ts
onChange(newDate.toLocaleString());
```
?